### PR TITLE
chore: setup o200k_base tokenizer

### LINKED ISSUE #42: Support New o200k_base
https://github.com/openai/tiktoken/commit/c0ba74c238d18b4824c25f3c27fc8698055b9a76

### PR DESCRIPTION
**Pull request by mbukeRepo**
This PR integrates the new tokenizer for gpt-4o `o200k_base`. 

Encoding file: 
[https://openaipublic.blob.core.windows.net/encodings/o200k_base.tiktoken 214](https://openaipublic.blob.core.windows.net/encodings/o200k_base.tiktoken)

Closes #42

### CODE REVIEW

File: README.md
Comment (caugner) [inline README.md:1]: You might also want to update lines 5 and 14:

https://github.com/niieani/gpt-tokenizer/blob/44ce38eae93d9a7b019691fb2baf0db97592d9e8/README.md#L5

https://github.com/niieani/gpt-tokenizer/blob/44ce38eae93d9a7b019691fb2baf0db97592d9e8/README.md#L14

Comment (mbukeRepo) [reply README.md:1]: Thanks Claas, I applied the requested changes.

**Comment by mccabejj** [comment]
Any idea when this will merge?

**Comment by prateekkathal** [comment]
@niieani @caugner Sry I am bumping this by tagging you.

We're trying to move to GPT-4o but the token calculations are coming out to be wrong since the base is different for GPT-4 Turbo & GPT-4o. It is imperative we have this PR merged as we are using this package in production. Until this PR gets merged, we are blocked.

**Review by niieani — APPROVED** [review]
Thanks!

**Comment by prateekkathal** [comment]
@niieani Thanks for merging. Seems like there is a formatting issue with the code blocking it from being published to NPM? 

Possible to check on your end and make these fixes?

**Pull request merged**